tests/test_docker_compose_snapshots.py

```
@@ -33,39 +32,37 @@ def tearDown(self):
-        self.assertTrue(self.wait_for_qdrant_ready())
+        assert result.returncode == 0
+        assert self.wait_for_qdrant_ready()
 
         # Check if container is running and accessible
         response = requests.get("http://localhost:6333/healthz", timeout=10)
-        self.assertEqual(response.status_code, 200)
+        assert response.status_code == 200
 
         # Verify volume mount exists
         inspect_result = subprocess.run(
             ["docker", "inspect", "test_qdrant_production", "--format={{.Mounts}}"],
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)